### ISSUE TITLE
HTML entities in CSS get converted in optimizer

### ISSUE THREAD

**Issue by bjalford** [ampproject/amp-toolbox#68]
e.g. > in CSS becomes &gt;

Can been seen on this page https://www.independent.co.uk/news/world/americas/donald-trump-tweets-red-hen-restaurant-sarah-huckabee-sanders-a8415711.html

Search for '&gt;' in the inline CSS

**Comment by andreban** [comment]
Hi Brian, 

Tried reproducing with this HTML:
```html
<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <link rel="canonical" href="self.html" />
  <meta name="viewport" content="width=device-width,minimum-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js"></script>
  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
  <style amp-custom>
    h1 {
      margin: 16px;
    }
    body > h1 {
      color: red;
    }
  </style>
</head>
<body>
  <h1>Hello, AMP world!</h1>
</body>
</html>
```
But the output is correct. Also tried with the URL mentioned above. Maybe a different config is causing this? Can you post the config that is being used?

**Comment by NickInSpace** [comment]
Hey Andre,

I can reproduce the issue I'm seeing with Brian, using the runtimeVersion demo and our config.

`
const express = require('express');
const path = require('path');
const app = express();
const AmpSsrMiddleware = require('../index.js');
const ampOptimizer = require('amp-toolbox-optimizer');
const runtimeVersion = require('amp-toolbox-runtime-version');

ampOptimizer.setConfig({
  transformers: [
    'AddAmpLink',
    'ServerSideRendering',
    'RemoveAmpAttribute',
    'AmpBoilerplateTransformer',
    'ReorderHeadTransformer',
    'RewriteAmpUrls',
    'GoogleFontsPreconnect'
  ]
});
const currentVersion = () => runtimeVersion.currentVersion();

const ssrMiddleware = AmpSsrMiddleware.create({
  ampOptimizer: ampOptimizer,
  runtimeVersion: currentVersion
});

app.use(ssrMiddleware);

const staticMiddleware = express.static(path.join(__dirname, '/public'));
app.use(staticMiddleware);

const DEFAULT_PORT = 3000;
const port = process.env.PORT || DEFAULT_PORT;
app.listen(port, () => {
  console.log('Example app listening on port 3000!');
});
`

**Comment by andreban** [comment]
I've been able to reproduce.
A few notes to confirm:
-  the error happens on the `amp-runtime` injection.
-  the `amp-custom` style remains correct.

This means the error only happens with runtimeVersion enabled, and injecting the runtime CSS.

**Comment by andreban** [comment]
Fixed and deployed on 0.3.1
